### PR TITLE
DO NOT MERGE: No hack/vendor.sh

### PR DESCRIPTION
**Pull request by runcom**
Fix https://github.com/projectatomic/skopeo/issues/131

Based on https://github.com/projectatomic/skopeo/pull/133 and missing #136, I'll rebase as needed to finally merge this.

This is using [https://github.com/Masterminds/glide](glide) and [https://github.com/sgotti/glide-vc](glide-vc) to do the vendoring w/o bothering ourself. Please read more on those repos but it's actually awesome and flexible (like updating just 1 dep instead of waiting for all of them to be merged, etc)

### CODE REVIEW

**Comment by runcom** [comment]
and to quote something really important - https://github.com/opencontainers/runc/pull/898#issuecomment-227924726

**Comment by runcom** [comment]
missing `oci` pkg and:
```
+- package: k8s.io/kubernetes
+  repo: https://github.com/openshift/kubernetes
+  version: 4a3f9c5b19c7ff804cbc1bf37a15c044ca5d2353
```
otherwise it panics (testing locally since Travis is so slow :))

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
+TEST
+
```
Comment (mtrmac) [inline README.md]: ?

**Comment by mtrmac** [comment]
In general I’m very much in favor of using standardized build infrastructure, especially if it is maintained by someone else and it works :)

The mention of `glide-vc` means that the workflow is not just a simple command, doesn’t it? If so, it would benefit from a `Makefile` target at least, if not some documentation (or at least a comment in that `Makefile`).

(I haven’t reviewed the `vendor` changes in detail; I assume that tests will tell us if something breaks.)

File: README.md (hunk 1 above)
Comment (runcom) [inline README.md]: testing stuff, will remove..

**Comment by runcom** [comment]
> The mention of glide-vc means that the workflow is not just a simple command, doesn’t it? If so, it would benefit from a Makefile target at least, if not some documentation (or at least a comment in that Makefile).

I already planned this and it will come shortly

**Pull request closed without merging**